drivers/include/net/netdev2.h

```diff
@@ -10,15 +10,153 @@
  */
 
 /**
- * @defgroup    drivers_netdev_netdev2 Generic Low-Level Network Driver Interface v2
+ * @defgroup    drivers_netdev_netdev2 Network Device Driver Interface
  * @ingroup     drivers_netdev
  * @brief       This is the second version of a generic low-level network
  *              driver interface
  * @{
  *
- * This interface is supposed to be a low-level interface for network drivers.
+ * @section sec_about About
```

Comment:
I have to admit that getting out "old" style doc can be tricky, because it can't be done purely in Markdown. But fine then we have mixed styles in the doc, better then no doc at all I guess :wink: